lib/medical_records/configuration.rb

```diff
@@ -21,24 +21,45 @@ def app_token
       Settings.mhv.medical_records.app_token
     end
 
+    ##
+    # @return [String] Base path for dependent URLs
+    #
+    def base_path
+      Settings.mhv.medical_records.host
+    end
+
     ##
     # @return [String] Service name to use in breakers and metrics
     #
     def service_name
       'MedicalRecords'
     end
 
-    # ##
-    # # Creates a connection with middleware for mapping errors, parsing XML, and
-    # # adding breakers functionality
-    # #
-    # # @see SM::Middleware::Response::SMParser
-    # # @return [Faraday::Connection] a Faraday connection instance
-    # #
-    # def connection
-    #   Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
-    #     conn.adapter Faraday.default_adapter
-    #   end
-    # end
+    ##
+    # Creates a connection
+    #
+    # @return [Faraday::Connection] a Faraday connection instance
+    #
+    def connection
+      Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
+        conn.use :breakers
+        # conn.request :camelcase
```

Comment:
Is this intentionally commented out?

Reply:
Currently I'm only using this connection for one endpoint (for authentication), and all further calls are being handled by our Ruby FHIR client. The camelcase actually breaks the authentication call because it modifies one of the params. I meant to remove it after validating that it worked.